packages/owl-bot/cloud-build/update-pr.yaml

```diff
@@ -67,7 +67,7 @@ steps:
     args: [
       '-l',
       '-c',
-      'git commit -m "🦉 Updates from OwlBot"; exit 0'
+      "git commit -m $'🦉 Updates from OwlBot\n\nSee https://github.com/googleapis/repo-automation-bots/blob/master/packages/owl-bot/README.md'; exit 0"
```

Comment:
I'm unfamiliar with what this `$` is doing.

Reply:
It's necessary for the \n\n to be interpreted as newlines.